### DISCUSSION TITLE
Cell component for URLs

### DISCUSSION THREAD

**Discussion by gappc** [noi-techpark/it.bz.opendatahub.databrowser#120] [Ideas]
While adding configs, @BlueRyse came up with the question if there is a dedicated cell component for URLs (see PR #118). At the moment there is no such component.

We should discuss if such a component is necessary for different view types and how it could look like. We should also discuss how a URL could be entered in the edit view.

**Comment by BlueRyse** [comment]
In the Webcams list view we show the Webcam URL, where you can see the latest live image from the Webcam.
It would be nice to have it as an Hypertext instead of a String.
Being able to directly click on the link opening a new page, would improve the user experience.

**Comment by BlueRyse** [comment]
Hello everyone,

@gappc We added a URL Cell Component in the latest PR, if you want to check it out.

The only missing thing, would be to highlight the text as an URL.

**Reply by ohnewein** [reply]
@BlueRyse this sounds interesting. Do you have a link on which I can test this?

**Reply by lukmay** [reply]
@ohnewein when the PR is merged you can find this Cell used in the [Testingmachine, Webcams, ListView ](https://databrowser.opendatahub.testingmachine.eu/dataset/table/tourism/v1/WebcamInfo) where we used this to display the Webcam URLs. 
(or build the project locally and test it there)
Hopefully this answers your question.

**Reply by ohnewein** [reply]
@lukmay thank you for the answer. I will wait until the PR will be merged and deployed on testingmachine. Thanks! 🙏

**Reply by gappc** [reply]
@lukmay: I added some comments on the PR #131, discussions welcome
